wallet-new/test/WalletNewJson.hs

```diff
@@ -0,0 +1,146 @@
+module WalletNewJson
```

Comment:
It's not a big deal, but I'm not sure I like calling this `WalletNewJson` since all of our other modules containing JSON round-trip and golden tests throughout the project have just been called `Json`.

I'm wondering if we should have a `Test.Cardano.Wallet.Json` or something.

Reply:
I would have however the module names in `wallet-new` don't follow that format. I was thinking a separate PR could reformat all of the names. Although it wouldn't hurt to implement the naming convention now.

Reply:
Oh, just saw @Jimbo4350's comment. I did notice that `wallet-new` is weirdly formatted. But it seems that we could just "clean as we go" and do it in this PR.